components/finalMessage/finalMessage.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react'
+
+import { Typography, Button } from '@mui/material'
+
+import { StyledFinalMessageContent, StyledTextArea } from '../../pageStyles/emailSender.styles'
```

Comment:
If we're using these components in two different files, let's move them to the common.ts file. Then import them from there in the emailSender.tsx and finalMessage.tsx!